vulture/utils.py

```diff
@@ -63,23 +65,32 @@ def get_decorator_name(decorator):
     return "@" + ".".join(reversed(parts))
 
 
-def get_modules(paths, toplevel=True):
+def get_modules(paths):
```

Comment:
How does the new code handle nested directories without recursion?

Reply:
The rglob() function does it.

Reply:
Ah, that's cool, didn't know about this function.